crates/forge/src/main.rs

```diff
@@ -36,7 +36,36 @@ fn default_fuzzer_runs() -> NonZeroU32 {
 }
 
 #[derive(Parser, Debug)]
-#[command(version)]
+#[command(
+    version,
+    help_template = "\
+{name} {version}
+{author-with-newline}{about-with-newline}
+Use -h for short descriptions and --help for more details.
+
+{before-help}{usage-heading} {usage}
+
+{all-args}{after-help}
+",
+    after_help = "Read the docs: https://foundry-rs.github.io/starknet-foundry/",
+    after_long_help = "\
+Read the docs:
+- Foundry Book: https://foundry-rs.github.io/starknet-foundry/
+- Cairo Book: https://book.cairo-lang.org/
+- Starknet Book: https://book.starknet.io/
+- Starknet Documentation: https://docs.starknet.io/
+- Scarb Documentation: https://docs.swmansion.com/scarb/docs.html
+
+Join the community:
+- Follow core developers on X: https://twitter.com/swmansionxyz
+- Chat on Telegram: https://t.me/starknet_foundry
+- Get support on Telegram: https://t.me/starknet_foundry_support
```

Comment:
Makes sense for the support group to be linked here, however maybe we could put it under some `Support` heading?

Would be more visible + probably less chance people go to the main Telegram group asking for support (as it's the first in order, above the support group)

Reply:
Would appreciate some suggestion here 🤔 
```
Engage on the contributors telegram channel:
Get support on the user support telegram channel:
```
wdyt?

Reply:
Perhaps sth like this?
```
- Get support via Telegram: https://t.me/starknet_foundry_support
- Or discord: https://discord.gg/KZWaFtPZJf
- Or join our general chat (Telegram): https://t.me/starknet_foundry
```

Reply:
Done